src/CppUTest/Utest.cpp

```diff
@@ -129,6 +129,14 @@ extern "C" {
 
 }
 
+/************************************************************************** */
+/* Define the two possible TestTerminators that can be selected at run time */
```

Comment:
Can you remove the comments. We consider comments to be mostly a smell, especially when they explain relative obvious things.